From 226a0f466c69a96d7b466a0bcefee697751a9a87 Mon Sep 17 00:00:00 2001
From: "Carsten Haitzler (Rasterman)" <raster@rasterman.com>
Date: Sat, 22 May 2021 13:59:44 +0100
Subject: [PATCH 06/13] alpm - fix apparent lack of braces in if clause

this clause on the same line does an emit then goto cont. this, by
the pattern about with if (alpm_list_find_str() != NULL) says thatwe
should continue if this is the case, so it seems we need some braces.
---
 backends/alpm/pk-alpm-sync.c | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/backends/alpm/pk-alpm-sync.c b/backends/alpm/pk-alpm-sync.c
index 005ba4ac2..c6f188075 100644
--- a/backends/alpm/pk-alpm-sync.c
+++ b/backends/alpm/pk-alpm-sync.c
@@ -30,6 +30,7 @@
 #include "pk-alpm-error.h"
 #include "pk-alpm-transaction.h"
 #include "pk-alpm-update.h"
+#include "pk-alpm-packages.h"
 
 static gboolean
 pk_alpm_transaction_sync_targets (PkBackendJob *job, const gchar **packages, gboolean update, GError **error)
@@ -70,8 +71,10 @@ pk_alpm_transaction_sync_targets (PkBackendJob *job, const gchar **packages, gbo
 
 			ignoregroups = alpm_option_get_ignoregroups (priv->alpm);
 			for (group_iter = alpm_pkg_get_groups (pkg); group_iter != NULL; group_iter = group_iter->next) {
-				if (alpm_list_find_str (ignoregroups, i->data) != NULL)
-					pk_alpm_pkg_emit (job, pkg, PK_INFO_ENUM_BLOCKED); goto cont;
+				if (alpm_list_find_str (ignoregroups, i->data) != NULL) {
+					pk_alpm_pkg_emit (job, pkg, PK_INFO_ENUM_BLOCKED);
+					goto cont;
+				}
 			}
 		}
 
-- 
2.31.1

